### PR TITLE
Add jwwandy to member.yaml

### PR DESCRIPTION
**Pull request by jwwandy**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/202)

### CODE REVIEW

File: members.yaml [hunk 1]
```diff
@@ -296,3 +296,10 @@
   lastName: Yan
   affiliation: Caicloud
   email: yanpenghui@caicloud.io
+
+- github: jwwandy
```
Comment (abhi-g) [inline members.yaml]: Hi Andy, We organize this file in alphabetical order of your github id. Can you please move this section up accordingly. thanks

Comment (jwwandy) [reply members.yaml]: Thanks for reminding me. Didn't notice that. Just move it to the correct position

**Pull request merged**